rb_vm_get_sourceline returns the wrong line number (unconditionally
set to 0) when the PC is 0.  This patch corrects that. 

The test for this is has to wait until 246-frame.patch, 
the code that implements pc_offset=  is done.

diff --git a/vm.c b/vm.c
index c919f5d..325bbad 100644
--- a/vm.c
+++ b/vm.c
@@ -774,6 +774,7 @@ rb_vm_get_sourceline(const rb_control_frame_t *cfp)
     if (RUBY_VM_NORMAL_ISEQ_P(iseq) && iseq->insn_info_size > 0) {
 	rb_num_t i;
 	size_t pos = cfp->pc - cfp->iseq->iseq_encoded;
+	line_no = iseq->insn_info_table[0].line_no;
 
 	if (iseq->insn_info_table[0].position == pos) goto found;
 	for (i = 1; i < iseq->insn_info_size; i++) {
